content/research_report/introductions.md

```diff
@@ -1,28 +1,25 @@
 # Introductions
 
-<!-- We all know denosing is important now. And we cannot understand brain activity without it. -->
+<!-- aim of the paragraph: We all know denosing is important now. And we cannot understand brain activity without it. -->
 Amongst functional magnetic resonance imaging (fMRI) acquisition protocols,
 resting state fMRI has become a tool of choice to explore human brain function in healthy and diseased populations {cite:p}`biswal_2010`.
 Functional connectivity {cite:p}`sporns_organization_2004`,
-a measure derived from resting state fMRI, is in particular a popular method to characterize the neural basis of <!-- any suggestions of some newer paper to cite is good. NC: Changed wording here due to repetition of 'choice', but you may prefer a different word to method -->
+a measure derived from resting state fMRI, is in particular a popular method to characterize the neural basis of <!-- any suggestions of some newer paper to cite is good. -->
 cognitive function {cite:p}`finn_functional_2015`,
 aging {cite:p}`andrews-hanna_disruption_2007`,
 and pathology {cite:p}`woodward_resting-state_2015`.
 The rise in popularity of resting state fMRI has also highlighted challenges and gaps in the analytic approach {cite:p}`cole_advances_2010`,
 such as distortion introduced by non-neuronal sources.
-These include head motion, scanner noise, cardiac and repiratory artifacts {cite:p`rogers_assessing_2007,murphy_2013`.
-Known as confounds or nuisance regressors, they pose a major challenge in functional connectivity analysis {cite:p`satterthwaite_impact_2012`.
+These include head motion, scanner noise, cardiac and repiratory artifacts {cite:p}`rogers_assessing_2007,murphy_2013`.
+Known as confounds or nuisance regressors, they pose a major challenge in functional connectivity analysis {cite:p}`satterthwaite_impact_2012`.
 For example, when functional connectivity is contaminated with motion, it can lead to altered network structure {cite:p}`power_scrubbing_2012`.
 Indeed, in younger subjects and neurologic/psychiatric patients the impact of head motion on connectivity is frequently reported {cite:p}`makowski_head_2019,satterthwaite_impact_2012`.
 Hence, reducing the impact of confounds, known as denoising,
 is a major component of the fMRI workflow to ensure the quality of analysis.
 
-<!-- Classes of nuisance regressors - like how load_confounds separate them -->
-<!-- need to add reference to this section-->
 ## Types of noise regressors
 
-<!-- NC: I thought the opening and closing sentences of this section were repeating the same informtion unecessarily -->
-<!-- so I combined them. It could also work at the end, but I think this way flows better to the next section -->
+<!-- aim of the paragraph: Classes of nuisance regressors - like how load_confounds separate them -->
```

Comment:
In this section I am describing these classes of regressors as unbiased as possible. Do we think it's necessary to add some comments?
For example, some for and against opinions on GSR, how tissue maps can effect the tissue signal and GSR, so on and so forth?

Reply:
I think that here you could point out that there is more too it (e.g. GSR literature), so the angle would be why it's important for users to make own decisions on regressors, but only a few sentences. Then you could come back to it in the discussion with more comments/description? That way it keeps the intro to-the-point.

Reply:
That sounds good and I can bridge to some of the point I would like to address. Also I only just realised I did not mention scrubbing at all...